app/Http/Requests/AttachStationsToPath.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Requests;
+
+class AttachStationsToPath extends BaseRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
```

Comment:
Maybe it would be a good idea to create an abstract class like `AuthorizedRequest` in your namespace and then extend it to avoid copypasting `authorize => true` evertywhere?

Reply:
@HerringTheCoder, please make this class abstract.